src/app/page.tsx

```diff
@@ -79,29 +18,24 @@ export default function TopicsPage() {
           <h2>Browse court forms by category</h2>
           <div className="row row-cols-1 row-cols-md-3 g-5">
             {legalTopics
-              .sort((a: Topic, b: Topic) => (a.priority < b.priority ? 1 : -1))
-              .filter((topic: Topic) => topic.always_visible)
-              .map((topic: Topic) => (
-                <TopicCard key={topic.codes[0]} topic={topic} />
+              .sort((a, b) => b.priority - a.priority)
+              .filter(
+                (topic) =>
+                  topic.always_visible ||
+                  (interviewsByTopic[topic.name] &&
+                    interviewsByTopic[topic.name].length > 0)
+              )
```

Comment:
LIST is a hierarchical code. 209A, which is FA-07-00-00-00, should go into the "Family" topic but it's getting sorted into Other

![image](https://github.com/SuffolkLITLab/courtformsonline.org/assets/7645641/f11bab16-965d-4900-a892-2888fbb8124a)

![image](https://github.com/SuffolkLITLab/courtformsonline.org/assets/7645641/0a8a80a5-e1ef-4563-b333-9c3a0d77214a)